### PR TITLE
Tdr 754 manually add series instructions

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: manual/add-new-series.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   * `[series_name]`: this is the series reference value, eg "LCO 72"
+   * `[series_description]`: this is the series reference value, eg "LCO 72"
+   
+4. Once the necessary scripts have been added to the new sql migration file follow the [migration deployment] instructions to deploy the changes to the database.
+    
+## Add new transferring body to Keycloak
+
+If the transferring body is new, and has not done a transfer through TDR, then it will need to be added to Keycloak.
+
+Follow the instructions for adding a [new transferring body to Keycloak].
```
Comment (suzannehamilton) [inline manual/add-new-series.md:48]: Probably worth making it clear that this needs to be done in production.

I also wonder if it's worth adding a step about testing the new transferring body and series in intg or staging? It's a manual test, but we won't be doing it very often. (And if we do have to do it a lot, that's when we should automate it!)

File: manual/add-new-series.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   
+3. Include a script in the new sql migration file adding the new series to the `Series` table:   
+   ```
+    INSERT INTO "Series" ("SeriesId", "BodyId", "Code", "Name", "Description") VALUES
+      (uuid_generate_v4(), [body_uuid], '[series_code]', '[series_name]', '[series_description]');
+   ```
+   * `[body_uuid]`: this is the value from the `BodyId` column in the `Body` table for the transferring body
+   * `[series_code]`: this is the series reference value, eg "LCO 72"
+   * `[series_name]`: this is the series reference value, eg "LCO 72"
+   * `[series_description]`: this is the series reference value, eg "LCO 72"
```
Comment (suzannehamilton) [inline manual/add-new-series.md:40]: Are series name, code and description always the same? Should we get rid of two of them?

Comment (suzannehamilton) [reply manual/add-new-series.md:50]: Yeah, that would be good. I think we've discussed it before, but I can't find any tasks to remove them.

**Review by suzannehamilton — APPROVED** [review]

File: manual/add-new-series.md (hunk 2 above)
Comment (TomJKing) [reply manual/add-new-series.md:50]: Added JIRA task: https://national-archives.atlassian.net/browse/TDR-1240

Comment (TomJKing) [reply manual/add-new-series.md:50]: Added JIRA task: https://national-archives.atlassian.net/browse/TDR-1240

**Pull request merged**